### PR TITLE
Nicolas/orc 130 orchestrator chainname panic

### PR DESCRIPTION
**Pull request by Kayanski**
This PR doesn't panic when using weird chain ids inside Mock

### CODE REVIEW

File: packages/cw-orch-mock/src/queriers/env.rs [hunk 1]
```diff
@@ -9,10 +9,20 @@ impl<A: Api, S: StateInterface> EnvironmentQuerier for MockBase<A, S> {
...
+            .rsplitn(2, '-')
+            .map(ToString::to_string)
+            .collect::<Vec<_>>();
+
+        // We make sure this doesn't error even if the chain id doesn't have the `osmosis-1` format
+        let chain_name = if chain_id_split.len() > 1 {
+            &chain_id_split[1]
+        } else {
+            &chain_id_split[0]
+        };
```
Comment (Buckram123) [inline packages/cw-orch-mock/src/queriers/env.rs:22]: won't it be simpler to just chain_id.rsplitn(2, '-').last().unwrap().to_string()

Comment (Buckram123) [reply packages/cw-orch-mock/src/queriers/env.rs:22]: it won't panic on unwrap even with "" chain_id

Comment (Kayanski) [reply packages/cw-orch-mock/src/queriers/env.rs:22]: Oh yeah, I forgot it has a max of 2 elements

File: packages/cw-orch-mock/src/queriers/env.rs [hunk 2]
```diff
@@ -9,10 +9,11 @@ impl<A: Api, S: StateInterface> EnvironmentQuerier for MockBase<A, S> {
     fn env_info(&self) -> EnvironmentInfo {
         let block_info = self.block_info().unwrap();
         let chain_id = block_info.chain_id.clone();
-        let chain_name = chain_id.rsplitn(2, '-').collect::<Vec<_>>()[1].to_string();
+        let chain_name = chain_id.rsplitn(2, '-').last().unwrap().to_string();
+
         EnvironmentInfo {
             chain_id,
-            chain_name,
+            chain_name: chain_name.clone(),
```
Comment (Buckram123) [inline packages/cw-orch-mock/src/queriers/env.rs:16]:
```suggestion
            chain_name,
```
No need to clone it anymore

**Review by Buckram123 — APPROVED** [review]
LGTM

**Pull request merged**